From c0b56c4fc0d6b0aa35961a1364754d8fddaa1bf5 Mon Sep 17 00:00:00 2001
From: DoloresPark <dolorespark@e-vertise.com>
Date: Wed, 8 Jul 2015 15:23:29 -0400
Subject: [PATCH] Revert "rearrange audio_policy structure to maintain
 compatibility with m470 blobs"

This reverts commit 0c2dd4b3634df2acb8f48015f7ac86cd1aa96e61.
This isn't needed when using the default audio policy module
(vs the proprietary 'audio_policy.tegra.so').
---
 include/hardware/audio_policy.h | 12 ++++--------
 1 file changed, 4 insertions(+), 8 deletions(-)

diff --git a/include/hardware/audio_policy.h b/include/hardware/audio_policy.h
index 5d0194a..99cb044 100644
--- a/include/hardware/audio_policy.h
+++ b/include/hardware/audio_policy.h
@@ -233,6 +233,10 @@ struct audio_policy {
             audio_stream_type_t stream,
             uint32_t in_past_ms);
 
+    bool (*is_stream_active_remotely)(const struct audio_policy *pol,
+            audio_stream_type_t stream,
+            uint32_t in_past_ms);
+
     bool (*is_source_active)(const struct audio_policy *pol,
             audio_source_t source);
 
@@ -242,14 +246,6 @@ struct audio_policy {
     /* check if offload is possible for given sample rate, bitrate, duration, ... */
     bool (*is_offload_supported)(const struct audio_policy *pol,
                                 const audio_offload_info_t *info);
-
-/* DoPa (20140401) - this method was added after 4.2.1 and isn't in the m470's
-                     'audio_policy.tegra.so' - move it to the end of the struct from
-                     its AOSP position after 'is_stream_active' to maintain compatibility
-*/
-    bool (*is_stream_active_remotely)(const struct audio_policy *pol,
-            audio_stream_type_t stream,
-            uint32_t in_past_ms);
 };
 
 
-- 
1.9.1

